src/com/owncloud/android/ui/activity/Uploader.java

```diff
@@ -424,6 +428,10 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
     private void populateDirectoryList() {
         setContentView(R.layout.uploader_layout);
+
+        mRadioBtnMoveFiles = (RadioButton) findViewById(R.id.uploader_radio_move);
+        mRadioBtnCopyFiles = (RadioButton) findViewById(R.id.uploader_radio_copy);
+        mRadioBtnCopyFiles.setChecked(true);
```

Comment:
Doesn't it override user decision when user de-selects this radio button and rotate the phone?